docs/subtensor-nodes/subtensor-storage-query-examples.md

```diff
@@ -552,7 +552,7 @@ title: "Subtensor Storage Query Examples"
 
     substrate = SubstrateInterface(url="ws://127.0.0.1:9945")
     netuid = 1
-    result = substrate.query('SubtensorModule', 'LastMechansimStepBlock', [netuid])
+    result = substrate.query('SubtensorModule', 'LastMechanismStepBlock', [netuid])
```

Comment:
Actually the one with the spello is correct. Turns out the developer pushed the function with this typo. `subtensorModule.lastMechansimStepBlock: u64`
See it here https://polkadot.js.org/apps/#/chainstate (Go Developer > Extrinsics and selector subtensorModule and the lastMechansimStepBlock function. So, please revert this change.

Reply:
Ok, thanks. Reverted with a comment